go/common/host/host.go

```diff
@@ -53,6 +53,6 @@ type StatsCollector interface {
 	// L2Recalc - called when a node has to discard the speculative work built on top of the winner of the gossip round.
 	L2Recalc(id gethcommon.Address)
 	NewBlock(block *types.Block)
-	NewRollup(node gethcommon.Address)
+	NewRollup(nodeIdx int)
```

Comment:
Did these gethcommon addresses just used to be like `0x00....0001`, `0x00...0002`?

I feel like nodeIdx isn't a thing that will exist in prod.

Maybe that's not worth worrying about for now, or maybe it should be nodeID or something? And that could be a type, that just aliases to int for now?